### ISSUE TITLE
Load common properties after calling ReadMetadata

### ISSUE THREAD

**Issue by taurenshaman** [drewnoakes/metadata-extractor-dotnet#278]
There are some commonly used properties for images, [ref](https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.imaging.bitmapmetadata?view=net-5.0):
* width, height, dpi
* Author
* Title, Comment, Keywords
* CameraManufacturer
* DateTaken
* Copyright
* Location
* DateTaken
* Format
* Subject
* Rating

So I hope there will be some helper method to get these properties from IReadOnlyList<Directory>.
Example code:
```
var directories = MetadataExtractor.ImageMetadataReader.ReadMetadata( fileStream );
ImageInfo info = XXXX.Load( directories ); 
// ImageInfo or other name is a struct including properties above.
// or
string author = MetadataExtractor.Utils.ImageInfo.GetAuthor( directories );
int width = MetadataExtractor.Utils.ImageInfo.GetWidth( directories );
```

Thank you very much ❤❤❤

**Comment by drewnoakes** [comment]
Thanks for your feature request. This is very similar in spirit to https://github.com/drewnoakes/metadata-extractor/issues/10.

**Comment by arnvanhoutte** [comment]
Yes please! I need to access the copyright tag in my image but the directory I need to look in depends on the extension of the image, which makes it the code really ugly

**Comment by vpenades** [comment]
I would like to know if there's been updated with this issue.

I think it's great to have an API so those that require it can browse into the details of the metadata... but if you just need some common properties, it's forcing everybody to reinvent the wheel into following the same steps to retrieve a given property.

maybe the solution would be to add a LINQ like API that would be implemented like this:

```c#
IEnumerable<Directory> directories = ImageMetadataReader.ReadMetadata(imagePath);

string title = directories.SelectTitles().FirstOrDefault();
string author = directories.SelectAuthors().FirstOrDefault();
GeoLocation loc = directories.SelectGeoLocations().FirstOrDefault();
```
